README.md

```diff
@@ -30,16 +30,12 @@ Some packages in otp-ui depend on sibling packages (e.g., `@opentripplanner/core
 
    > "@opentripplanner/package-to-test": **"file:../package-to-test"**
 
-2. In your source files, find and replace:
-
-   > opentripplanner/package-to-test/**lib**/types
-
-   with
+2. Run:
+   `yarn && yarn dev`
 
-   > opentripplanner/package-to-test/**src**/types
+### Storyshot testing
 
-3. Run:
-   `yarn && yarn dev`
+This repo utilizes the [Storyshot](https://storybook.js.org/docs/react/workflows/snapshot-testing) Storybook addon to perform snapshot tests of every story in this monorepo. Whenever the script `yarn unit` is ran, the Storyshot addon will be included along with all the other tests. It will compare the output of every story to the saved snapshot of that story. This provides a quick way to make sure nothing drastic has changed and that every single story is able to render without an error. However, often times these snapshots will need to be updated and that can be accomplished by running `yarn unit -u`.
```

Comment:
One thing that should be mentioned is that the snapshots are for the contents initially shown when the user initially opens a story, and that the snapshots do not cover changes from the user interacting with each story. See for instance the `base-map` stories that show empty `<div>` tags, or in the `trip-details` package where snapshot don't cover the expandable calories description.

Reply:
Improved this documentation in 1d369aea05797ecbe500da148cec00a439ec60e1.